### PR TITLE
Blog post for the Sprint #162

### PR DESCRIPTION
**Pull request by danidoni**

### CODE REVIEW

File: _posts/development/2024-04-08-improvements-on-the-redesigned-request-page.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+### Showing The Reason a Review Was Asked in the Review Popup
+
+When someone asks for a review, they can leave a reason for it. The review popup now shows what the review creator entered as the reason for the review.
+
+![](/images/posts/2024-04-08/reason_for_a_review.png)
+
+### Don't Collapse the Message About Why the Request Was Declined
+
+When someone declines a request they usually leave a message explaining why. Until now that reason was collapsed by default and that made a bit hard to understand by the request creator why that request was declined. This is now shown by default.
```
Comment (bmwiedemann) [inline _posts/development/2024-04-08-improvements-on-the-redesigned-request-page.md:27]: "that made a bit hard" => "that made it a bit hard"

Comment (danidoni) [reply _posts/development/2024-04-08-improvements-on-the-redesigned-request-page.md:27]: I think the "it" is "why that request was declined", so the "it" is not needed there

File: _posts/development/2024-04-08-improvements-on-the-redesigned-request-page.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: Improvements On the Redesigned Request Page
+category: development
+---
+
+So, we're back after Easter Holidays! In this blog post we're going to see some improvements we've been working on lately on the redesigned Request page.
```
Comment (saraycp) [inline _posts/development/2024-04-08-improvements-on-the-redesigned-request-page.md:7]:
```suggestion
So, we're back after the Easter Holidays! In this blog post, we're going to see some improvements we've been working on lately on the redesigned Request page.
```

File: _posts/development/2024-04-08-improvements-on-the-redesigned-request-page.md (hunk 1 above)
Comment (saraycp) [reply _posts/development/2024-04-08-improvements-on-the-redesigned-request-page.md:27]: I was about to correct the same. This is another proposal:

```suggestion
When someone declines a request they usually leave a message explaining why. That message was collapsed by default until now, making it harder for the request creator to understand why that request was declined. This is now shown by default.
```

File: _posts/development/2024-04-08-improvements-on-the-redesigned-request-page.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+{% include partials/_series-of-posts-about-request-redesign.md %}
+
+### Defer Loading Content in the Request Show Page
+
+Users trying to look into the changes of Submit Requests had to wait for the whole content to load before seeing anything. When those changes were big, they had to wait a lot. 
+
+We deferred the loading of the content of the individual file changes so now it appears to load faster.
```
Comment (saraycp) [inline _posts/development/2024-04-08-improvements-on-the-redesigned-request-page.md:15]: I wouldn't say the goal is "appearing it's loading faster". The goal is that the user sees something in the Conversation (and other tabs) ASAP. And thanks to the spinner they know they have to wait for the changes and they don't think something is wrong/not loading.

File: _posts/development/2024-04-08-improvements-on-the-redesigned-request-page.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+![](/images/posts/2024-04-08/defer_content_loading.gif)
+
+### Showing The Reason a Review Was Asked in the Review Popup
```
Comment (saraycp) [inline _posts/development/2024-04-08-improvements-on-the-redesigned-request-page.md:19]:
```suggestion
### Showing The Reason Why a Review Was Asked in the Review Popup
```

**Review by saraycp — CHANGES_REQUESTED** [review]

File: _posts/development/2024-04-08-improvements-on-the-redesigned-request-page.md (hunk 2 above)
Comment (danidoni) [reply _posts/development/2024-04-08-improvements-on-the-redesigned-request-page.md:7]: I'm sorry, but everywhere I look "Easter Holiday" is not preceded by "the"...

File: _posts/development/2024-04-08-improvements-on-the-redesigned-request-page.md (hunk 3 above)
Comment (danidoni) [reply _posts/development/2024-04-08-improvements-on-the-redesigned-request-page.md:15]: Alright, what about something like "We deferred the loading of the content of the file changes so now users don't have to wait too much before something appears on the page"?

File: _posts/development/2024-04-08-improvements-on-the-redesigned-request-page.md (hunk 2 above)
Comment (saraycp) [reply _posts/development/2024-04-08-improvements-on-the-redesigned-request-page.md:7]: Ok! The ignore my suggestion. No problem.

File: _posts/development/2024-04-08-improvements-on-the-redesigned-request-page.md (hunk 3 above)
Comment (saraycp) [reply _posts/development/2024-04-08-improvements-on-the-redesigned-request-page.md:15]: Perfect!!

File: _includes/partials/_series-of-posts-about-request-redesign.md [hunk 5]
```diff
@@ -10,4 +10,4 @@ In July 2023 we introduced [Set Bugowner action and diff comments for non-beta u
 In September 2023 we focused on [Build Results Summary Gets Into Shape](/2023/09/08/request-workflow-redesign).
 Later 2023, in November, we added the [Build Status Refresh and Other Insights on Our Request Page](/2023/11/30/introducing-build-status-refresh-and-other-insights-in-our-request-page).
 The year 2024 started with [Revamping the Request Build Status Page and Introducing the Dark Mode](/2024/01/25/revamping-the-build-status-page-and-introducing-the-dark-mode).
-This time we give you a little update with [Build Results Summary Chart Links to Build Results Overview](/2024/02/19/build-results-summary-chart-links-to-build-results).
+We continued in February with [Build Results Summary Chart Links to Build Results Overview](/2024/02/19/build-results-summary-chart-links-to-build-results).
```
Comment (saraycp) [inline _includes/partials/_series-of-posts-about-request-redesign.md:13]: Mentioning the current blog post is missing.

Comment (danidoni) [reply _includes/partials/_series-of-posts-about-request-redesign.md:13]: Done

**Review by saraycp — APPROVED** [review]
LGTM, please don't forget to squash the commits.

**Pull request merged**